### ISSUE TITLE
Package for conda and/or pip

### ISSUE THREAD

**Issue by awvwgk** [ZimmermanGroup/pyGSM#27]
Is there interest to package this project for conda-forge or upload on PyPI? I could certainly help with creating a conda package.

**Comment by craldaz** [comment]
Yes, I am very interested in doing this, I definitely need to make it pip installable but having a conda package would be great. Any help would be appreciated.

Also, I think the current master branch is pretty stable right now, I should probably tag it and/or work on a dev branch to refrain from breaking things for other people. 

Thanks!

**Comment by awvwgk** [comment]
A tagged release would be much appreciated for packaging on conda-forge.
